run_pipeline.sh

```
@@ -36,76 +37,90 @@ yamllint -c .yamllint.yaml .github/workflows/*.yml
 SERVER_PID=$!
 
-# Wait for server to be ready
 for i in {1..20}; do
   if curl -f http://localhost:$SERVER_PORT/index.html >/dev/null 2>&1; then
     echo "Web server ready"
     break
   fi
   sleep 1
 done
+
 if [ $i -eq 20 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)